index.html

```diff
@@ -28,23 +28,24 @@
   <body>
     <header class="nav">
       <a class="logo" href="#">Bwalya Nonde</a>
-      <nav class="nav-buttons">
-        <ul class="nav-li">
-          <li><a>Portfolio</a></li>
-          <li><a>Contact</a></li>
-          <li><a>About</a></li>
-          <li>
-            <i class="far fa-envelope"></i>
-          </li>
-        </ul>
-      </nav>
-      <div class="bars">
-        <i class="fas fa-bars bars"></i>
+      <div id="myNav" class="overlay">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+        <div class="overlay-content">
+          <a href="javascript:void(0);" onclick="menuOptions('#portfolio')">Portfolio<i class="fa fa-angle-right"></i></a>
+          <a href="javascript:void(0);" onclick="menuOptions('#about')">About<i class="fa fa-angle-right"></i></a>
+          <a href="javascript:void(0);" onclick="menuOptions('#contact-us')">Contact<i class="fa fa-angle-right"></i></a>
+        </div>
+      </div>
```

Comment:
- [ ] You did a good job on the project functionalities but there is a little issue, When I click on the portfolio I am being redirected to the about section, and when I click on the about I am being redirected to the projects section. Please fix this issue.